packages/ui-shared/src/components/ButtonLinkBox/ButtonLinkBox.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { cnCreate, Button, TextLink } from '@megafon/ui-core';
+import './ButtonLinkBox.less';
+
+export interface IButtonLinkBoxProps {
+    /** Custom className */
+    className?: string;
+    /** Button title */
+    buttonTitle?: string;
+    /** Button url */
+    buttonUrl?: string;
+    /** Button color */
+    buttonColor?: 'green' | 'purple';
+    /** Link title */
+    linkTitle?: string;
+    /** Link url */
+    linkUrl?: string;
+    /** Horizontal alignment */
+    hAlign?: 'center';
```

Comment:
Может не будем делать пропсы с перечислением, в котором отсутствует дефолтное значение?

```suggestion
    hAlign?: 'left' | 'center';
```

Логика в том, что если в обертке нужно будет также динамически определять вид выравнивания, то нужно будет делать что-то типа

```
const props = isAlignCenter ? { hAlign: 'center' } : undefined
<ButtonLinkBox {...props}/>
```

вместо

```
<ButtonLinkBox hAlign={isAlignCenter ? 'center' : 'left'}/>
```

Т.е. api компонента становится более "определенным".

Это же относится к компоненту TitleDescriptionBox, не заметил перед влитием.